.husky/pre-commit

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd "$(dirname -- "$0")/../."
+yarn build
```

Comment:
I am worried that running `yarn build` during the commit process will interfere with fixing bugs. Do we really need this? What are we solving by running the build here?

Reply:
Making sure that the dist build is always up to date this is a similar structure to other deque actions